site/app/controllers/AbstractController.php

```diff
@@ -129,7 +129,7 @@ protected function tryGetMark(Component $component, string $mark_id, bool $rende
      * @param bool $render_json true to render a JSEND response to the output in the failure/error case
      * @return string|bool false in the fail/error case
      */
-    protected function tryGetSubmitterIdFromAnonId(string $anon_id, bool $render_json = true) {
+    protected function tryGetSubmitterIdFromAnonId(string $anon_id, string $g_id = null, bool $render_json = true) {
```

Comment:
Would there be a case where we want g_id as null?

Reply:
Currently, there is no such place, so, default value has been removed.